theme/lumo/vaadin-grid-tree-toggle-styles.html

```diff
@@ -67,6 +67,31 @@
         background-size: var(--vaadin-grid-tree-toggle-level-offset) var(--vaadin-grid-tree-toggle-level-offset);
         background-position: calc(var(--vaadin-grid-tree-toggle-level-offset) / 2 - 2px) 0;
       }
+
+      /* RTL specific styles */
+
+      :host([dir="rtl"]) {
+        margin-left: 0;
+        margin-right: calc(var(--lumo-space-s) * -1);
+      }
+
+      :host([dir="rtl"]) [part="toggle"] {
+        margin-left: 0;
```

Comment:
`margin-right` is not reset here. It could be reset or `margin-right` could be changed inside a `:host(:not([dir="rtl"])) [part="toggle"]` selector.